modules/test/playwright/fixtures/FDSPages.fixture.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {test} from '@playwright/test';
+
+import {FDSPage} from '../pages/FDS/FDS.page';
```

Comment:
FDS should be named to something more standard, like the name of the OSGi bundle (as in product-navigation-....)

Reply:
Also, we should follow the same file case (ie: fds.page.ts as in applicationMenu.page.ts)

Reply:
yes, I suggest to use `DataSetManager` or `DSM`.
FDS is reserved for automations concerning frontend data set only (if we happen to create them later on)

Reply:
It would be `dataSetManager`, then (first in lowercase).

However, for the `applicationsMenu` we have `product-navigation-applications-menu` which is quite different from `dataSetManager` because:

1. It is snake case
2. data set manager is not the name of any OSGi bundle

So either we rename our thing to something like frontend-dataset (because of frontend-data-set-web) or we propose to name all playwright pages in a different way.

Pages are shared between everybody, so we cannot follow different conventions per team.

Reply:
You guys have the experience of naming conventions, so I could only humbly fulfill your wishes

Reply:
I'm the guy that puts quick names in this stuff. 
I like `dataSetManager` (first in lowercase).